azure_monitor/readme.md

```diff
@@ -1,6 +1,34 @@
-# Getting Started with App Insights for Python via OpenCensus Azure monitor exporters
+---
+title: 'Getting Started with App Insights for Python via OpenCensus Samples'
+description: Getting Started with App Insights for Python via OpenCensus Samples.
+author: givenscj
+ms.author: givenscj
+ms.devlang: python
+ms.topic: sample
+ms.date: 07/27/2022
+ms.custom: []
+---
 
-This procedure configures your Python application to send telemetry to the Application Insights feature of the Azure Monitor service with the OpenCensus Azure monitor exporters.  It works with any Python application on-premises or in the cloud.
+# Getting Started with App Insights for Python via OpenCensus Samples
+
+Many of the samples in this folder support the https://docs.microsoft.com/en-us/azure/azure-monitor/app/opencensus-python articles.  Reference these articles for more information on how OpenCensus works with Python and Application Insights.
+
+The following samples are available in this repo:
```

Comment:
List mixes
- Azure Service Integration (e.g., Functions)
- 3 Pillars (Logging, Tracing, Metrics)
- Web Handlers (Flask, Django)

Can we make more consistent?

Reply:
Not really sure what to do with this comment.  Azure App Service is Flask/Django and Functions.  All of them mix some aspect of logging, tracing and metrics.  Don't know what "consistent" means in this context.

Reply:
Ok, added some details to the sample apps.